abr_control/arms/jaco2/config.py

```diff
@@ -76,16 +78,16 @@ def __init__(self, hand_attached=False, **kwargs):
 
         # inertia values in VREP are divided by mass, account for that here
         self._M_LINKS = [
-            sp.diag(0.5, 0.5, 0.5, 0.02, 0.02, 0.02),  # link0
-            sp.diag(0.5, 0.5, 0.5, 0.02, 0.02, 0.02),  # link1
-            sp.diag(0.5, 0.5, 0.5, 0.02, 0.02, 0.02),  # link2
-            sp.diag(0.5, 0.5, 0.5, 0.02, 0.02, 0.02),  # link3
-            sp.diag(0.5, 0.5, 0.5, 0.02, 0.02, 0.02),  # link3
-            sp.diag(0.5, 0.5, 0.5, 0.02, 0.02, 0.02),  # link4
-            sp.diag(0.25, 0.25, 0.25, 0.01, 0.01, 0.01)]  # link5
+            sp.diag(0.5, 0.5, 0.5, 0.1, 0.1, 0.1),  # link0
```

Comment:
I imagine your values are better here, I didn't spend any time tuning the M matrix for the sim